### DISCUSSION TITLE
Consider archiving some apps after the transition to lessen the workload

### DISCUSSION THREAD

**Discussion by Cwpute** [FossifyOrg/General-Discussion#42] [Ideas]
Having so many apps to maintain is a huge undertaking, especially after having lost our main contributor.

To alleviate this, i suggest we should consider letting go of some of the apps from the suite, which are among the lesser used, less critical ones, and that require the most specific skills for the least impactful result. But most importantly, there should be foss altenatives to these apps that show dedicated maintenance.

Would you think this as a reasonable consideration, and a possibility ?

**Comment by Aga-C** [comment]
It was already discussed here: https://github.com/FossifyOrg/General-Discussion/issues/1.

**Reply by Cwpute** [reply]
Ah, sorry, didn't see it in open suggestions and my mind defaulted to "it just hasn't been discussed yet".
I'll close this then!

**Comment by Cwpute** [comment]
Apps i'd consider for archiving:
* Calculator: on F-droid alone, there are a wide range of maintained alternatives from simple to specialized calculatons that are more efficient, more lightweight etc… Bugs in calculator apps can be [hard to spot](https://www.windowslatest.com/2018/04/05/microsoft-fixes-a-long-standing-windows-calculator-bug-in-latest-windows-10-build-17639/) and stupidly dangerous.
* Flashlight: unreasonably heavy app that as countless alternatives on F-droid
* Keyboard: extremely important app on smartphones, that needs thoughtful maintenance, and which is easily superseded by AnysoftKeyboard's existence
* Music Player: again, an outstanding number of FOSS alternatives exist, maintained by passionate people, with either distinctive or generalist purposes.
* Simple Thank You: easily replaced with donation links in the about page and welcome pages of every other app.

Most of the other apps are fine imo. Maybe arguments can be made against maitaining the Camera app but i don't know much about it.
